hosting/static-website/README.md

```diff
@@ -0,0 +1,128 @@
+# Static website
+The example shows how to deploy a simple, static website on the IC in an asset canister. The website is very simple, it just shows the Dfinity logo and has a basic styling of the logo. 
+
+![Website](README_images/website.png)
+
+## Introduction
+The purpose of this example is to show how to deploy a static website in an asset canister. While the website in this example is very simple, the method is the same for more advanced static website, e.g. based on popular static site generators.
+
+This example covers:
+
+- Build a very simple HTML website
+- Create the dfx.json file
+- Deploy the canister smart contract locally
+- Test the frontend in browser 
+
+## Installation
+This example project can be cloned, installed and deployed locally, for learning and testing purposes. The instructions are based on running the example on either macOS or Linux, but when using WSL2 on Windows, the instructions will be the same.
+
+### Prerequisites
+The example project requeres the following installed:
+
+- GitHub
+- DFX 
+
+GitHub can be installed from various package managers. DFX can be installed following the instructions [here](https://smartcontracts.org/docs/quickstart/local-quickstart.html#download-and-install).
+
+### Install
+Install the example dapp project:
+
+```bash
+$ git clone https://github.com/dfinity/examples
+$ cd motoko/static-website
+```
+
+## Documentation
+There are two parts in this dapp, one is the website and the other is the `dfx.json`. Since there's no backend, there is not any benefit of using the `dfx new project_name` to setup a template, the `dfx.json` file is all there is needed.
```

Comment:
(corrected run-on sentences and grammar)
```suggestion
There are two parts in this dapp. One is the website and the other is the `dfx.json`. Since there's no backend, there is not any benefit of using the `dfx new project_name` to setup a template. The `dfx.json` file is all that is needed.
```